src/main/java/it/smartcommunitylab/aac/webauthn/persistence/WebAuthnUserAccount.java

```diff
@@ -0,0 +1,135 @@
+package it.smartcommunitylab.aac.webauthn.persistence;
+
+import java.util.Set;
+
+import javax.persistence.Column;
+import javax.persistence.Entity;
+import javax.persistence.EntityListeners;
+import javax.persistence.FetchType;
+import javax.persistence.GeneratedValue;
+import javax.persistence.Id;
+import javax.persistence.OneToMany;
+import javax.persistence.Table;
+import javax.persistence.UniqueConstraint;
+import javax.validation.constraints.NotNull;
+
+import com.yubico.webauthn.data.ByteArray;
+
+import org.springframework.data.jpa.domain.support.AuditingEntityListener;
+
+import it.smartcommunitylab.aac.SystemKeys;
+import it.smartcommunitylab.aac.core.model.UserAccount;
+
+@Entity
+@Table(name = "webauthn_users", uniqueConstraints = @UniqueConstraint(columnNames = { "provider_id", "username" }))
+@EntityListeners(AuditingEntityListener.class)
+public class WebAuthnUserAccount implements UserAccount {
+
+    private static final long serialVersionUID = SystemKeys.AAC_CORE_SERIAL_VERSION;
+
+    @Id
+    @GeneratedValue
+    private Long id;
+
+    // TODO: civts, use converter
+    @Column(unique = true)
```

Comment:
userHandle is globally unique?

Reply:
The user handles are probabilistically unique: we use random sequences of 64 bytes, as [recommended](https://www.w3.org/TR/webauthn/#sctn-user-handle-privacy) by the WebAuthn specification.
The current code assumes that the user handle is unique and uses it to globally identify a WebAuthn account.

Reply:
Closed with 680db77030e4c6be96631a9ff413aac401808ef1